Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2335,6 +2337,11 @@ Color RenderStyle::colorByApplyingColorFilter(const Color& color) const
     return transformedColor;
 }
 
+Color RenderStyle::resolvedColorAndColorFilter(const StyleColor& color) const
```

Comment:
probably nitpicky about naming here, but `resolvedColorAndColorFilter` sounds like we're resolving the -apple-color-filter property.

`resolveColorAndApplyColorFilter` (what Antti suggested) or `resolveColorWithColorFilter` make more sense here.

Reply:
`resolveColorWithColorFilter` is more confusing than `resolvedColorAndColorFilter` to me.
I will let the name as is for this PR, feel free to change it in another one if it's worth it.